pycbc/transforms.py

```diff
@@ -257,34 +248,67 @@ def _convert_inverse(self, maps):
         -------
         out : dict
             A dict with key as parameter name and value as numpy.array or float
-            of converted values.
+            of transformed values.
         """
         sx, sy, sz = self._outputs
         data = coordinates.cartesian_to_spherical(maps[sx], maps[sy], maps[sz])
         out = {param : val for param, val in zip(self._outputs, data)}
         return self.format_output(maps, out)
 
+
+class CartesianSpin1ToSphericalSpin1(SphericalSpin1ToCartesianSpin1):
+    """The inverse of SphericalSpin1ToCartesianSpin1.
+    """
+    name = "cartesian_spin_1_to_spherical_spin_1"
+    inverse = SphericalSpin1ToCartesianSpin1
+    _inputs = inverse._outputs
+    _outputs = inverse._inputs
+    transform = inverse.inverse_transform
+    inverse_transform = inverse.transform
+    jacobian = inverse.inverse_jacobian
+    inverse_jacobian = inverse.jacobian
+
+
+SphericalSpin1ToCartesianSpin1.inverse = CartesianSpin1ToSphericalSpin1
```

Comment:
Ok, done. I've moved all of the inverse transforms to their own section, and then grouped these lines into a single block.